### PR TITLE
Show a taxon's icon when it is set

### PR DESCRIPTION
**Pull request by cee-dub**
If there's an icon set for the Taxon, the admin form should show it.

### CODE REVIEW

**Comment by cbrunsdon** [comment]
Makes sense to me, :+1:

**Comment by cee-dub** [comment]
Great! No rush to merge this, I'm overriding it locally for now.

Could maybe use some CSS? I'm not good at that.

**Comment by cbrunsdon** [comment]
@cee-dub no worries, glad to hear you're overriding it locally.

We wait for a second thumbs up from a core member before we merge. Generally if its an improvement we'll merge it even if its "rough". CSS could probably make this better? But since its an improvement overall so its likely to be accepted as it is and we can deal with the intricacies of the UI later.

Thanks though! All improvements are super appreciated, no matter the level of polish.

File: backend/app/views/spree/admin/taxons/_form.html.erb [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <%= f.field_container :icon do %>
       <%= f.label :icon %><br />
       <%= f.file_field :icon %>
+      <%= image_tag f.object.icon(:small) if f.object.icon.present? %>
```
Comment (jhawthorn) [inline backend/app/views/spree/admin/taxons/_form.html.erb]: `:small` isn't a style on icon by default.

We could use `:mini` or `:normal`

**Comment by cee-dub** [comment]
I guess I'd prefer `:mini`, since that's default.

**Comment by jhawthorn** [comment]
:+1: Thanks @cee-dub

**Pull request merged**